### DISCUSSION TITLE
Unhandled Rejection (LoginFailedError): Failed to login: Invalid authentication response.

### DISCUSSION THREAD

**Discussion by vitorsiqueirarecife** [blockstack/connect#242] [Q&A]
The app of my constumer in production enviroment was fall down, when the user try to use the login modal. 

Note: with some users happen the problem, but other working very well.

App: https://preaviz.altius.com/  

Module used: @stacks/connect-react

Error:
![image](https://user-images.githubusercontent.com/58955708/172481660-36ff5e3d-0387-4c60-8b85-6ad1c473794a.png)

**Comment by vitorsiqueirarecife** [comment]
@friedger

**Comment by friedger** [comment]
@vitorsiqueirarecife Which version of stacks/connect do you use?

**Reply by vitorsiqueirarecife** [reply]
In production i am using the old version, but i tried use with latest version and the problem happen too.

![image](https://user-images.githubusercontent.com/58955708/172619640-ee930be3-2c86-4027-9f79-1e5240b39f0a.png)

**Reply by kyranjamie** [reply]
Looks like `@blockstack/connect` is a stale dependency, could that be causing the problem?

**Reply by vitorsiqueirarecife** [reply]
Should i remove that?

**Reply by kyranjamie** [reply]
Yes

**Reply by vitorsiqueirarecife** [reply]
Other problem happen, maybe because i just removed the dependence `@blockstack/connect`. What dependence i need put instead that?

<img width="992" alt="image" src="https://user-images.githubusercontent.com/58955708/172725830-fd9b3fe1-c463-45f5-87b9-7716891614f6.png">

<img width="886" alt="image" src="https://user-images.githubusercontent.com/58955708/172725769-2e80624c-2c7f-4361-9f23-2f282ad57cd7.png">

**Comment by friedger** [comment]
Maybe @pradel has an idea about what happens to the names?

**Reply by pradel** [reply]
Hum, when the wallet stopped injecting names I didn't see this issue anymore, don't know what's happening there
